projects/packages/masterbar/src/admin-menu/class-base-admin-menu.php

```diff
@@ -580,12 +587,16 @@ public function dashboard_switcher_scripts() {
 				var viewLink = $( '#view-link' );
 				var viewWrap = $( '#view-wrap' );
 
-				viewLink.on( 'click', function() {
+				// Remove any existing click handlers before adding a new one
+				viewLink.off( 'click' ).on( 'click', function() {
+					console.log( 'click' );
 					viewWrap.toggle();
 					viewLink.toggleClass( 'screen-meta-active' );
+					console.log( 'class is toggled' );
```

Comment:
Also here.

Reply:
Indeed. I was still testing, you beat me to it :) Done in d3eb69a12d4564aeeeccea437e54e6890df0edd5